grace/training/train.py

```diff
@@ -215,5 +222,32 @@ def valid(loader):
         # Print out the logging string:
         LOGGER.info(logger_string)
```

Comment:
could we add a line break at the end of each epoch string? this will make the outputs easier to read.

Reply:
Raising a new issue from this to further customise the logger in the `styling.py` script.